src/App.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import { v4 } from 'uuid';
+import './App.css';
+
+class App extends Component {
+  state = {
+    tempDescription: '',
+    todos: [
+      {
+        id: v4(),
+        description: 'delectus aut autem',
+        completed: false,
+      },
+      {
+        id: v4(),
+        description: 'quis ut nam facilis et officia qui',
+        completed: false,
+      },
+      {
+        id: v4(),
+        description: 'fugiat veniam minus',
+        completed: false,
+      },
+      {
+        id: v4(),
+        description: 'et porro tempora',
+        completed: true,
+      },
+    ],
+  };
+
+  handleSubmit = event => {
+    event.preventDefault(); //not sure why I had to add this but read about it online - it fixed an issue with the todo not appearing on the screen
+    const todo = {
+      id: v4(),
+      description: this.state.tempDescription,
+      completed: false,
+    };
+    this.setState({
+      todos: [todo, ...this.state.todos],
```

Comment:
Do we want to add the new todo to the top of the list, or the bottom?

Reply:
I tried both ways and left it at the top, as when it was at the bottom, the input kept getting pushed down everytime a new todo was added. Maybe I can move the input up to the top and then add the todo's at the bottom...?